src/ORM/Table.php

```diff
@@ -121,6 +121,13 @@ class Table implements RepositoryInterface, EventListenerInterface
     use EventManagerTrait;
 
     /**
+     * Name of default validation set.
+     *
+     * @var string
+     */
+    const VALIDATOR_DEFAULT = 'default';
```

Comment:
Noted. You got a point but for some reason, I still prefer the `DEFAULT_VALIDATOR` over `VALIDATOR_DEFAULT`. 

What do others think?